src/main/java/org/mozilla/jss/ssl/javax/JSSEngineReferenceImpl.java

```diff
@@ -1638,10 +1660,10 @@ private void cleanupSSLFD() {
         if (!closed_fd && ssl_fd != null) {
             try {
                 SSL.RemoveCallbacks(ssl_fd);
-                PR.Close(ssl_fd);
```

Comment:
My understanding at the time of writing this was that `PR.Close(...)` freed all memory at the NSS SSL PRFileDesc wrapper level, and which in turned freed the underlying transit PRFileDesc (here, our circular buffer PRFileDesc wrapper) as well. 

By removing `PR.Close(ssl_fd)`, but adding `super.(...)` to `ssl_fd.close()`, you effectively [do the same thing](https://github.com/dogtagpki/jss/blob/master/src/main/java/org/mozilla/jss/nss/PRFDProxy.java#L10), just in a different order. I'm not really sure it matters much.